README.md

```diff
@@ -1,15 +1,32 @@
 # device-setup
 
-To setup an Abigail device for the first time, clone the `abigail-device`, `wifi-setup`, and `user-setup` repositories and execute the setup scripts. Use the following commands:
+## Hardware setup
 
-```
+The project is guaranteed to run with the following hardware:
+* a Raspberry 3 with Raspbian Jessie-lite
+* a MPU-6050 accelerometer ([look how to connect on your Pi](http://blog.bitify.co.uk/2013/11/interfacing-raspberry-pi-and-mpu-6050.html))
+* [Jabra conference speaker](http://www.jabra.com/Business/speakerphones/jabra-speak-series/jabra-speak-510)
+
+It should work also OK with some differences: Raspberry Pi 2 with a WiFi dongle,
+other accelerometers or other microphones/speakers.
+
+## Software setup
+
+To setup an Abigail device for the first time, on the device clone the
+[`abigail-device`](https://github.com/project-abigail/abigail-device/),
+[`wifi-setup`](https://github.com/project-abigail/wifi-device/)
+repositories and execute the setup scripts.
+
+Use the following commands:
+
+```shell
 sudo apt-get install -y git
 git clone https://github.com/project-abigail/abigail-device
 git clone https://github.com/project-abigail/wifi-setup
-./abigail-device/bootstrap-scripts/raspberry-pi-setup.sh
-./wifi-setup/setup-scripts/raspberry-pi-setup.sh
-cd abigail-device; npm install; cd ..
-cd wifi-setup; npm install; cd ..
+sudo ./abigail-device/bootstrap-scripts/raspberry-pi-setup.sh
+sudo ./wifi-setup/setup-scripts/raspberry-pi-setup.sh
+(cd abigail-device && npm install)
+(cd wifi-setup && npm install)
```

Comment:
it's more idiomatic, a parens will start a subshell so after the subshell quits we're back at the same directory